tests/data/csv/pacbio_library_batch.csv

```diff
@@ -0,0 +1,3 @@
+Source,Tag,Template prep kit box barcode,Volume,Concentration,Insert Size
```

Comment:
It would still be good to find a way we don't have to use csv files.

Reply:
I have now removed the CSV files, as they are now generated as part of the factory